bundle/cert-manager-operator/0.15.0-alpha.0/cert-manager-operator.v0.15.0-alpha.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,90 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    description: "x509 certificate management for Kubernetes"
+    createdAt: "2020-04-20T12:00:00Z"
+    support: "cert-manager team"
+    alm-examples: ' [{"apiVersion":"operator.cert-manager.io/v1alpha1","kind":"CertManager","metadata":{"name":"cert-manager","namespace":"cert-manager"},"spec":{}}]'
+    capabilities: Full Lifecycle
+    categories: Security
+    certified: 'false'
+    containerImage: quay.io/jetstack/cert-manager-operator:v0.15.0-alpha.0
+    repository: https://github.com/jetstack/cert-manager
+  name: cert-manager-operator.v0.15.0-alpha.0
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - kind: CertManager
+      displayName: CertManager
+      name: certmanagers.operator.cert-manager.io
+      version: v1alpha1
+      description: 'Represents the cert-manager installation'
+  description: 'This is where the description goes'
```

Comment:
nit: maybe take first paragraph from the website?

Reply:
whoops...